support_bundle/get_support_bundle.sh

```diff
@@ -480,6 +480,12 @@ main() {
     echo "Fetching the sysdigcloud-config ConfigMap"
     kubectl ${KUBE_OPTS} get configmap sysdigcloud-config -o yaml | grep -v password | grep -v apiVersion > ${LOG_DIR}/config.yaml || true
 
+    # Collect the sysdigcloud-values secret, and write to the log directory if the backend version is 6.5 or higher
+    if [[ "$BACKEND_VERSION" =~ ^(6.5) ]]; then       
```

Comment:
this will only pick up versions that start with 6.5

Reply:
Hey @mbreitung I've corrected the issue as suggested whenever you can take a look 🙏